src/Controller/EnMarche/ProcurationController.php

```diff
@@ -99,18 +97,21 @@ public function profileAction(Request $request): Response
      */
     public function electionsAction(Request $request): Response
     {
-        $command = $this->getProcurationFlow()->getCurrentModel();
+        $procurationSession = $this->get(ProcurationRequestSession::class);
+        $command = $procurationSession->getCurrentModel();
         $command->recaptcha = (string) $request->request->get('g-recaptcha-response');
 
-        $form = $this->createForm(ProcurationElectionsType::class, $command);
-        $form->handleRequest($request);
+        $form = $this->createForm(ProcurationElectionsType::class, $command)
+            ->handleRequest($request)
+        ;
 
         if ($form->isSubmitted() && $form->isValid()) {
             $manager = $this->getDoctrine()->getManager();
+
             $manager->persist($command);
             $manager->flush();
-
-            $this->getProcurationFlow()->reset();
+            // clean
```

Comment:
I don't see the point of this comment. Why the session is clean-up would be a useful comment though.

Reply:
Changed. Is it better?